code/__DEFINES/traits.dm

```diff
@@ -236,6 +236,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NANITE_MONITORING	"nanite_monitoring" //The mob's nanites are sending a monitoring signal visible on diag HUD
 /// Prevents mob from riding mobs when buckled onto something
 #define TRAIT_CANT_RIDE			"cant_ride"
+#define TRAIT_BLOODY_MESS		"bloody_mess" //from heparin, makes open bleeding wounds rapidly spill more blood
+#define TRAIT_COAGULATING		"coagulating" //from coagulant reagents, this doesn't affect the bleeding itself but does affect the bleed warning messages
```

Comment:
> this doesn't affect the bleeding itself but does affect the bleed warning messages

Is this comment true? It looks like it manipulates the flow of blood too.

Reply:
The actual effects on bleeding and wounds that coagulant has is handled in the coagulant reagent. Unlike TRAIT_BLOODY_MESS, which is used to increase bleeding rates in wounds in multiple places, this trait itself is only used as a sign that someone has coagulant in their blood for bleed warning messages.